Source/FieldSolver/SpectralSolver/SpectralAlgorithms/PMLPsatdAlgorithm.cpp

```diff
@@ -81,6 +86,56 @@ PMLPsatdAlgorithm::pushSpectralFields(SpectralFieldData& f) const {
             const Complex Bzx = fields(i,j,k,Idx::Bzx);
             const Complex Bzy = fields(i,j,k,Idx::Bzy);
 
+            Complex Ex, Ey, Ez;
+            Complex Bx, By, Bz;
+
+            // Used only if dive_cleaning = true and divb_cleaning = true
+            Complex Exx, Eyy, Ezz;
+            Complex Bxx, Byy, Bzz;
+            Complex Fx, Fy, Fz;
+            Complex Gx, Gy, Gz;
+            Complex F, G;
+
+            if (!dive_cleaning && !divb_cleaning)
+            {
+                Ex = Exy + Exz;
+                Ey = Eyx + Eyz;
+                Ez = Ezx + Ezy;
+
+                Bx = Bxy + Bxz;
+                By = Byx + Byz;
+                Bz = Bzx + Bzy;
+            }
+            else if (dive_cleaning && divb_cleaning)
```

Comment:
If we don't think that we will ever need dive_cleaning != divb_cleaning, we may consider to use only one `dive_divb_cleaning` variable